action.php

```diff
@@ -638,6 +639,7 @@ protected function load_template() {
             '@WIKI@'    => $conf['title'],
             '@WIKIURL@' => DOKU_URL,
             '@DATE@'    => dformat(time()),
+            '@USERNAME@'=> $INFO['userinfo']['name'],
```

Comment:
```suggestion
            '@USERNAME@'=> $INFO['userinfo']['name'] ?? '',
```
This entry could be not set, please add the isset check